rest/route/github.go

```diff
@@ -667,6 +683,36 @@ func (gh *githubHookApi) requireSigned(ctx context.Context, userRepo data.UserRe
 	return nil
 }
 
+func (gh *githubHookApi) requireApprovals(ctx context.Context, userRepo data.UserRepoInfo, prNum, requiredApprovalCount int) error {
+	settings, err := evergreen.GetConfig()
```

Comment:
Instead of calling `evergreen.GetConfig()`, can we use `evergreen.GetEnvironment().Settings()` and pass the admin settings into this function? The reasoning is that we don't want to overly rely on loading the settings directly from the DB unless there's a specific reason to do so.

Reply:
switched to evergreen.GetEnvironment().Settings().GetGithubOauthToken()